cloudweatherreport/cloud_weather_report.py

```diff
@@ -195,7 +195,7 @@ def main(args, test_plan):
             continue
         last_successful_status = status
         benchmark_results = []
-        if test_plan.get('benchmark'):
+        if status is not None and test_plan.get('benchmark'):
```

Comment:
Error in `test_results` doesn't always mean we don't want to run benchmarks. The unit tests can fail and generate exceptions. So we can't really just check error in `test_results`. 

If `test_result` and `status` is None, we don't want to executing the line of code you mentioned.